src/main/kotlin/no/nav/syfo/client/HttpClientCommon.kt

```diff
@@ -16,6 +17,20 @@ fun httpClientDefault() = HttpClient(CIO) {
     expectSuccess = true
 }
 
+fun httpClientWithRetry(
+    numberOfRetries: Int = 3,
```

Comment:
Holde kanskje med 1(evt 2) retries?

Når det gjelder `retryOnException`, går det ann å konfigurere så den ikke gjør retry ved 4xx-feil?

Reply:
Ja, det er gode muligheter for å konfigurere dette, men jeg er litt usikker på hva som vil fungere best her. Ved ClosedReceiveChannelException får man vel egentlig ingen respons. 

Jeg tenkte at vi kunne prøve det ut litt på denne måten, så skjønner vi kanskje etter hvert hvordan det er lurest å bruke det.